odbc/src/api/functions.rs

```diff
@@ -769,16 +767,49 @@ pub unsafe extern "C" fn SQLDescribeCol(
 #[no_mangle]
 pub unsafe extern "C" fn SQLDescribeColW(
     hstmt: HStmt,
-    _col_number: USmallInt,
-    _col_name: *mut WChar,
-    _buffer_length: SmallInt,
-    _name_length: *mut SmallInt,
-    _data_type: *mut SqlDataType,
-    _col_size: *mut ULen,
-    _decimal_digits: *mut SmallInt,
-    _nullable: *mut Nullability,
+    col_number: USmallInt,
+    col_name: *mut WChar,
+    buffer_length: SmallInt,
+    name_length: *mut SmallInt,
+    data_type: *mut SqlDataType,
+    col_size: *mut ULen,
+    decimal_digits: *mut SmallInt,
+    nullable: *mut Nullability,
 ) -> SqlReturn {
-    unimpl!(hstmt);
+    panic_safe_exec!(
+        || {
+            let stmt_handle = MongoHandleRef::from(hstmt);
+            stmt_handle.clear_diagnostics();
```

Comment:
I actually am not sure if we should clear the errors here. Should we, instead, let users clear the errors by retrieving them? @nbagnard?

Reply:
Makes one wonder how you can get more than one Rec to necessitate the ability to retrieve them by number

Reply:
I think it's mainly if you had warnings and kept going. It could be more than one message if the function ends with success_with_info. Because, you are right, otherwise it sounds impossible.